src/test/kotlin/org/opensearch/replication/integ/rest/UpdateAutoFollowPatternIT.kt

```diff
@@ -102,8 +98,6 @@ class UpdateAutoFollowPatternIT: MultiClusterRestTestCase() {
             }, 60, TimeUnit.SECONDS)
         } finally {
             followerClient.deleteAutoFollowPattern(connectionAlias, indexPatternName)
```

Comment:
As a followup, we can also cleanup the autofollow patterns after test.
We can use auto-follow-stats for fetching the autofollow rules.